server/queue/queue-rabbitmq/src/test/java/org/apache/james/queue/rabbitmq/RabbitMQMailQueueTest.java

```diff
@@ -329,6 +332,51 @@ void dequeueShouldRetryLoadingErrors(CassandraCluster cassandra) throws Exceptio
                 .extracting(item -> item.getMail().getName())
                 .containsExactly(name1, name2, name3);
         }
+
+        @Test
+        void dequeueShouldNotRetryLoadingErrors(CassandraCluster cassandra) throws Exception {
+            String name1 = "myMail1";
+            String name2 = "myMail2";
+            String name3 = "myMail3";
+
+            getMailQueue().enQueue(defaultMail()
+                .name(name1)
+                .build());
+
+            getMailQueue().enQueue(defaultMail()
+                .name(name2)
+                .build());
+
+            getMailQueue().enQueue(defaultMail()
+                .name(name3)
+                .build());
+
+            cassandra.getConf().registerScenario(returnEmpty()
+                .forever()
+                .whenQueryStartsWith("SELECT * FROM blobs WHERE id=:id;"));
+
+            ConcurrentLinkedDeque<String> dequeuedNames = new ConcurrentLinkedDeque<>();
+            Flux.from(getMailQueue().deQueue())
+                .take(3)
+                .doOnNext(item -> dequeuedNames.add(item.getMail().getName()))
+                .doOnNext(Throwing.consumer(item -> item.done(true)))
+                .subscribeOn(Schedulers.elastic())
+                .subscribe();
+
+            // One second should be enough to attempt dequeues while we fail to load blobs
+            Thread.sleep(1000);
+
+            // Restore normal behaviour
+            cassandra.getConf().registerScenario(returnEmpty()
```

Comment:
Why is it the same as declared in the previous lines ?

Reply:
Sorry it executeNormally